### ISSUE TITLE
validation set

### ISSUE THREAD

**Issue by tututututurana** [facebookresearch/sam2#417]
In the project’s YAML file, I noticed the configuration for the training set, but I’m currently unclear on how to set up the validation set. In the context of your code framework, could you suggest the correct way to add and configure the validation set? Would it be possible to provide an example?

**Comment by rgtent** [comment]
Hello, is the problem solved, I also have the same problem, stuck for a long time
